src/Resources/Pieces.php

```diff
@@ -2,11 +2,25 @@
 
 namespace Mvdnbrk\DhlParcel\Resources;
 
+use Mvdnbrk\DhlParcel\Contracts\Arrayable;
 use Mvdnbrk\DhlParcel\Exceptions\ResourceNotAccepted;
 use Tightenco\Collect\Support\Collection;
 
-class Pieces extends BaseResourceCollection
+class Pieces extends Collection implements Arrayable
 {
```

Comment:
`Tightenco\Collect\Support\Collection` already implements it's own `Arrayable` interface, so I think this can be removed.

Reply:
Please rename the class to `PiecesCollection`. To be clear this is a collection of items.